Codeinwp/neve
# Fix global colors not applying on button text

### Summary
Some global colors didn't have `!important` and the color was not applying

### Will affect the visual aspect of the product
NO

### Test instructions
- Create a  page and add a button component
- Set the button as primary and set its text color to Neve - Dark Background
-  Check the button on the frontend ( nothing happens, the text color is still white )
- Add this fix and check the button text color again
